.github/pull_request_template.md

```diff
@@ -1,8 +1,10 @@
+https://github.com/DataBiosphere/azul/issues/NUMBER
+
 Author
 
 - [ ] PR title references issue
 - [ ] Title of main commit references issue
-- [ ] PR is linked to Zenhub issue
+- [ ] PR is connected to Zenhub issue and issue description links to issue
```

Comment:
```suggestion
- [ ] PR is connected to Zenhub issue and link added to PR description
```

Reply:
Clearly what I have written here is wrong, but I mis-copied [what @hannes-ucsc had requested](https://github.com/DataBiosphere/azul/pull/3223#discussion_r671403159).

Since your suggestion and his say essentially the same thing, I'd rather defer to his. If there's a problem with this that I'm missing, please let me know. Sorry for the trouble, hopefully this should be a straightforward review.